src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/NameEntity.kt

```diff
@@ -41,25 +47,32 @@ class AliasEntity(
   @Column(name = "date_of_birth")
   val dateOfBirth: LocalDate? = null,
 
+  @Column
+  @Enumerated(STRING)
+  val type: NameType,
+
   @Version
   var version: Int = 0,
 ) {
   companion object {
-    private fun from(alias: Alias): AliasEntity? =
+    private fun from(name: Name): NameEntity? =
       when {
-        isAliasPresent(alias.firstName, alias.middleNames, alias.lastName) ->
-          AliasEntity(
-            firstName = alias.firstName,
-            middleNames = alias.middleNames,
-            lastName = alias.lastName,
+        isAliasPresent(name.title, name.firstName, name.middleNames, name.lastName) ->
```

Comment:
This method should be renamed to `nameExists` or something

Reply:
Done :+1